generator/model.go

```diff
@@ -837,9 +837,12 @@ func (sg *schemaGenContext) buildProperties() error {
 			emprop.GenSchema.ValueExpression += "()"
 		}
 
+		emprop.GenSchema.Extensions = emprop.Schema.Extensions
+
 		// set custom serializer tag
 		if customTag, found := emprop.Schema.Extensions[xGoCustomTag]; found {
 			emprop.GenSchema.CustomTag = customTag.(string)
+			delete(emprop.GenSchema.Extensions, xGoCustomTag)
```

Comment:
@liber-takano I merged this without paying much attention at first.
Now that I am looking at it more at ease, I am really wondering why you want to delete this.
How about other template users who use Extensions?
Why is the redundant presence of xGoCustomTag here an issue for you?

Reply:
For xGoCustomTag, it is okay to make your judgment top priority.